extensions/wikia/AdEngine/js/video/player/porvata/floaterConfiguration.js

```diff
@@ -43,7 +40,9 @@ define('ext.wikia.adEngine.video.player.porvata.floaterConfiguration', [
 			onBeforeDetach: function (floatingContext) {
 				var viewport = floatingContext.elements.viewport;
 
-				viewport.style.height = viewport.offsetHeight + 'px';
+				// Magic 14 is related to #INCONTENT_WRAPPER > #INCONTENT_PLAYER margins
+				// and prevents jumping when video is going to float
+				viewport.style.height = (viewport.offsetHeight - 14) + 'px';
```

Comment:
Maybe we could calculate `14` based on DOM node properties (margins in this case)?

Reply:
It would require using `getComputedStyle` - is it worth?

Reply:
Nein! Let's leave it as it is. Comment is enough.